sources/HUBViewControllerImplementation.m

```diff
@@ -173,8 +173,11 @@ - (void)dealloc
 
 - (void)loadView
 {
-    self.view = [[HUBContainerView alloc] initWithFrame:CGRectZero];
+    HUBContainerView *hubContainerView = [[HUBContainerView alloc] initWithFrame:CGRectZero];
+    self.view = hubContainerView;
+
     [self createCollectionViewIfNeeded];
+    hubContainerView.collectionView = self.collectionView;
```

Comment:
Move this into `createCollectionViewIfNeeded` instead? That way, when the collection view is re-created (for example after a memory warning), the container view's reference will be updated.